packages/babel-plugin-transform-es2015-parameters/test/fixtures/parameters/default-rest/actual.js

```diff
@@ -0,0 +1,15 @@
+const a = 1;
+function rest(b = a, ...a) {
```

Comment:
When I run this example on node 6
```js
const a = 1;
function rest(b = a, ...a) {
console.log(b, a);
}

rest();
```

this is the result:
```js
[] []
```

Not sure if v8 is spec conform, but I would assume it. So `a` is not undefined but an empty array, when nothing to `rest`. And the default params seem to account the rest param.